.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules/
```

Comment:
@codewylle only in the root, man?

Reply:
I usually have .gitignore for each package